.github/workflows/run_pr_tests.yml

```diff
@@ -381,3 +391,35 @@ jobs:
         toolchain_file: "scripts/../platform/arm-linux/aarch64-toolchain.cmake"
         extra_flags: -DCA_BUILTINS_TOOLS_DIR=${{ github.workspace }}/llvm_install_native/bin
 
+  # Based on: mr-ubuntu-gcc-x86_64-clik
+  run-ubuntu-gcc-x86_64-clik:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 60
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@v4.1.0
+    - name: setup-ubuntu
+      uses: ./.github/actions/setup_build
+      with:
+        llvm_version: '19'
+        llvm_build_type: RelAssert
+        save:  ${{ inputs.update_cache }}
+    - run: echo WORKSPACE is $GITHUB_WORKSPACE && echo PWD is `pwd` && ls -al
+    - run: cmake -Bbuild_clik -GNinja -DCMAKE_INSTALL_PREFIX=install clik
+    - run: LD_PRELOAD=/lib/x86_64-linux-gnu/libpthread.so.0 ninja -Cbuild_clik check
+
+  # Based on: mr-ubuntu-gcc-x86_64-clik-refsi
+  run-ubuntu-gcc-x86_64-clik-refsi:
```

Comment:
just a thought since the start overheads seem bigger than the actual run-time, maybe just combine into one job (but two different steps).

Reply:
OK - I'll do that. Jobs are green with just a ninja install.

Reply:
Jobs combined